docs/workbench/README.md

```diff
@@ -34,6 +41,7 @@ cd submarine
 | JAVA_HOME           | Set your java home path, default is `java`.                  |
 | WORKBENCH_JAVA_OPTS | Set the JAVA OPTS parameter when the Workbench process starts. If you need to debug the Workbench process, you can set it to `-agentlib:jdwp=transport=dt_socket, server=y,suspend=n,address=5005` |
 | WORKBENCH_MEM       | Set the java memory parameter when the Workbench process starts. |
+| MYSQL_VERSION       | The version of mysql jdbc jar would be downloaded. The default value is 5.1.39. |
```

Comment:
No need to let the user configure it. Specify the version directly in the `workbench-daemon.sh` shell script.